src/components/SearchBar/SearchBar.jsx

```diff
@@ -92,8 +92,13 @@ class SearchBar extends Component {
     if (!!dateFilters.startDate.value) {
       params = `start=${this.state.dateFilters.startDate.value}`; 
     }
+
+    if (!!dateFilters.startDate.value && !!dateFilters.endDate.value) {
```

Comment:
what's the point of using double bang here? 

it's not different than just using 

dateFilters.startDate.value && dateFilters.endDate.value

Reply:
that doesn't make sense, because `(dateFilters.startDate.value && dateFilters.endDate.value)` ensures a boolean value on its own

Reply:
in that case, we'll need to be consistent everywhere else and remove redundant double bang